manuscript/manuscript.Rmd

```diff
@@ -491,68 +491,51 @@ Many of our most realistic ecological models exist only as numerical simulations
 and can easily be extended into more complex problems once RL agents can master the basics.
 Finding acceptable policies even in relatively simple problems may prove beyond the reach of existing algorithms and available computing power.
-<!-- Marcus is working on tightening up these next few paragraphs, precisely which issues actually get highlighted TBD-->
-In seeking either to adopt or assess a deep RL-based approach to an ecological issue, we highlight three key issues that are important to bear in mind: instability, interpretability, and reward design.
-
-
-<!-- reason 1: instability-->
-<!-- ML: I think this needs to be tightened up a bit-->
-One leading challenge for deep RL today is the problem of instability.
-RL agents trained on narrowly defined environments may perform poorly even when the environment is altered in seemingly trivial way, such as a re-scaling of parameters.
-To avoid this, researchers should seek to both train and evaluate agents against a range of environments representing different parameters or processes dynamics.
-Stable performance can be challenging even when using the same environment.
-For a range of reasons, including high variance [@cite] and the exploration vs. exploitation trade-off [@cite], model-free and model-based algorithms tend to have unstable performance on environments with non-tabular state-action spaces [cite RL that matters].
-For instance, it is not uncommon to observe vastly different behavior when an agent is initialized with different random seeds [cite ibid].
-Such unstable behavior can often be avoided by alternate selection of hyperparameters, so researchers should consider stability as well as performance during hyperparameter selection.
-The choice of RL algorithm, design of the reward function, and duration of training can also influence stability.
-
+In seeking either to adopt or assess a deep RL-based approach to an ecological issue, we highlight three key issues that are important to bear in mind: generalizability, interpretability, and reward design.
+
+<!-- reason 1: generalizability-->
+<!-- ML: pivoting this to generalizability but still rewrote past paragraph which might fit in the appendix somewhere-->
+Currently, deep RL agents struggle to generalize their skills [@pmlr-v97-cobbe19a].
+This inability can affect performance during training and at evaluation. 
+For instance, training an agent on an environment that demands learning an assortment of skills is typically much more difficult than training an agent to learn one specific skill [@teh_distral_2017].
+And, at evaluation, if you give an agent a novel task, including tasks that are closely related to the examples that the agent saw in training, the agent can easily fail to generalize its learned behavior to this new situation [@pmlr-v97-cobbe19a].
+These failures are all due to the brittleness of deep neural networks. 
```

Comment:
"all" might be a little strong? Isn't there an aspect of this that is due to the nature of a given environment than the network alone?

Reply:
Yeah, I intended 'all' as a qualifier on 'these', not as a modifier on 'due', will change this.